.circleci/config.yml

```diff
@@ -70,6 +70,51 @@ jobs:
       - clear_environment:
           cache_key: snarkos-stable-cache
 
+  storage:
+    docker:
+      - image: cimg/rust:1.56.1
```

Comment:
note: the current stable version is `1.57.0`, we might want to update that one as well